include/SampleClip.h

```diff
@@ -77,9 +77,9 @@ class SampleClip : public Clip
 
 	bool isPlaying() const;
 	void setIsPlaying(bool isPlaying);
+	void setSampleBuffer(std::unique_ptr<SampleBuffer>&& sb);
```

Comment:
Yeah, probably. Just figured that an r-value reference would read better. I'll change it.

Also, one thing to mention is that I had to move this declaration outside of the slots category because the Qt moc for this file was trying to call the copy constructor for the ``unique_ptr``. Just in case if you were wondering.